### PR TITLE
Remove unused X11 dep

### PR DESCRIPTION
**Pull request by adjamhub**
Working to free lxqt from x11extras.... the first one is trivial :)

### CODE REVIEW

**Comment by adjamhub** [comment]
ping

**Review by tsujan — APPROVED** [review]
GTM

File: src/batteryinfoframe.cpp [hunk 1]
```diff
@@ -62,8 +62,8 @@ BatteryInfoFrame::~BatteryInfoFrame()
 void BatteryInfoFrame::onBatteryChanged()
 {
     mUi->stateValue->setText(BatteryHelper::stateToString(mBattery->chargeState()));
-    mUi->energyFullValue->setText(QString::fromLatin1("%1 Wh (%2 %)").arg(mBattery->energyFull(), 0, 'f', 2).arg(mBattery->capacity()));
-    mUi->energyValue->setText(QString::fromLatin1("%1 Wh (%2 %)").arg(mBattery->energy(), 0, 'f', 2).arg(mBattery->chargePercent()));
+    mUi->energyFullValue->setText(QString::fromLatin1("%1 Wh (%2 %)").arg( mBattery->energyFull(), mBattery->capacity(), 0, 'f', 2 ));
+    mUi->energyValue->setText(QString::fromLatin1("%1 Wh (%2 %)").arg(mBattery->energy(), mBattery->chargePercent(), 0, 'f', 2));
```
Comment (tsujan) [inline src/batteryinfoframe.cpp:66]: Oh, my bad! These are incorrect. I wonder how many instances I missed in other places. Should check all of them :(

**Review by tsujan — CHANGES_REQUESTED** [review]

File: src/batteryinfoframe.cpp (hunk 1 above)
Comment (adjamhub) [reply src/batteryinfoframe.cpp:66]: Not sure I understood the problem here...

Comment (tsujan) [reply src/batteryinfoframe.cpp:66]: See https://doc.qt.io/qt-5/qstring.html#arg-14 (and uts following methods). Here we have numbers, not strings.

Comment (adjamhub) [reply src/batteryinfoframe.cpp:66]: oh... now I see. Sorry. Reverting those lines anche checking the other ones.

**Review by tsujan — APPROVED** [review]
GTM now.

**Pull request merged**